dynamo/configuration.py

```diff
@@ -4,16 +4,32 @@
 from cycler import cycler
 import matplotlib.pyplot as plt
 
-# set the data store mode.
-# saving memory or storing more results
-# modes: verbose, succint
-data_store_mode = "verbose"
-keep_filtered_genes = True
-keep_raw_layers = True
-
-if data_store_mode == "succint":
-    keep_filtered_genes = False
-    keep_raw_layers = False
+
+class DynamoConfig:
+    # set the data store mode.
+    # saving memory or storing more results
+    # modes: verbose, succint
+    data_store_mode = "verbose"
+    keep_filtered_genes_default = True
+    keep_raw_layers_default = True
+    keep_filtered_cells_default = True
```

Comment:
I said you also configure the parameters for tl.dynamics in this. 

del_moment in tl.dynamics (double confirm the argument) should be set too

Reply:
it is the del_2nd_moments argument